applications/GeoMechanicsApplication/tests/test_settlement_workflow/ProjectParameters_stage1.json

```diff
@@ -39,7 +39,7 @@
         "rayleigh_m":                         0.0,
         "rayleigh_k":                         0.0,
         "strategy_type":                      "newton_raphson",
-        "convergence_criterion":              "displacement_criterion",
+        "convergence_criterion":              "residual_criterion",
```

Comment:
This is to make sure the residual_criterion is also covered. Indeed, the test failed when changing the criterion, which got fixed after removing the 'IsConverged' call

Reply:
Just as a reminder: I would propose to add a small comment to this `json` file which explains why there is one deviating `"convergence_criterion"` (just for our future selfs ;-))